### PR TITLE
Add color icons to lazygit

### PR DESCRIPTION
**Pull request by aashish2057**
- **PR Description**
PR that updates lazygit to have file icons with colors, some existing icons have been changed to be better matched for colors as well. Issue #2968 
- **Please check if the PR fulfills these requirements**

* [x] Cheatsheets are up-to-date (run `go generate ./...`)
* [x] Code has been formatted (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#code-formatting))
* [x] Tests have been added/updated (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/integration/README.md) for the integration test guide)
* [x] Text is internationalised (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#internationalisation))
* [x] Docs (specifically `docs/Config.md`) have been updated if necessary
* [x] You've read through your own file changes for silly mistakes etc

### CODE REVIEW

File: pkg/gui/presentation/icons/file_icons.go [hunk 1]
```diff
@@ -5,327 +5,327 @@ import (
...
+	".ps1":            {"\ue795", 241},     // 
+	".psd":            {"\ue7b8", 241},     // 
+	".pxm":            {"\uf1c5", 241},     // 
+	".py":             {"\ue606", 214},     // 
+	".pyc":            {"\ue606", 214},     // 
+	".r":              {"\uf25d", 241},     // 
+	".rakefile":       {"\ue21e", 241},     // 
+	".rar":            {"\uf410", 241},     // 
+	".razor":          {"\uf1fa", 241},     // 
+	".rb":             {"\ue21e", 241},     // 
```
Comment (jesseduffield) [inline pkg/gui/presentation/icons/file_icons.go:241]: There's quite a few greys here. Is there anywhere online with a more complete list of colours?

**Comment by jesseduffield** [comment]
Looks like we've got some structs being converted to strings incorrectly:

image

File: pkg/gui/presentation/icons/file_icons.go (hunk 1 above)
Comment (aashish2057) [reply pkg/gui/presentation/icons/file_icons.go]: I  changed some more of them, the list I was working from didn't have many of the icons lazygit supports, I went through trying to match the colors to there relevant logos for the rest of them, some will have to be grey. I also fixed the structs being converted to string. @jesseduffield

**Comment by jesseduffield** [comment]
@aashish2057 I've made some changes but wasn't able to push them to your branch (because it's your fork's master branch). So I've instead raised another PR and I've merged that.

Thanks for your contribution!

**Pull request closed without merging**